src/chainparams.cpp

```diff
@@ -100,8 +100,8 @@ class CMainParams : public CChainParams {
         assert(maxUint/UintToArith256(consensus.powLimit) >= consensus.nPowAveragingWindow);
         consensus.nPowMaxAdjustDown = 32; // 32% adjustment down
         consensus.nPowMaxAdjustUp = 16; // 16% adjustment up
-        consensus.nPreBlossomPowTargetSpacing = Consensus::PRE_BLOSSOM_POW_TARGET_SPACING;
-        consensus.nPostBlossomPowTargetSpacing = Consensus::POST_BLOSSOM_POW_TARGET_SPACING;
+        consensus.nPreBlossomPowTargetSpacing = Consensus::PRE_BLOSSOM_POW_EXPECTED_SECS_PER_BLOCK;
+        consensus.nPostBlossomPowTargetSpacing = Consensus::POST_BLOSSOM_POW_EXPECTED_SECS_PER_BLOCK;
```

Comment:
I would prefer not to make this change since "{Pre,Post}BlossomPoWTargetSpacing" is what the spec calls these values (section 5.3). You could use, e.g. `PRE_BLOSSOM_POW_TARGET_SPACING_SECS`.

Reply:
Alright, I will revert this.